samples/drivers/jesd216/boards/nrf52840dk_nrf52840_spi.overlay

```diff
@@ -0,0 +1,50 @@
+/*
+ * Copyright (c) 2022 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+/* This DTS overlay allows to test MX25R6435F  using SPI_NOR driver */
+
+&pinctrl {
+	spi1_alt_default: spi1_alt_default {
+		group1 {
+			psels = <NRF_PSEL(SPIM_SCK, 0, 19)>,
+				<NRF_PSEL(SPIM_MOSI, 0, 20)>,
+				<NRF_PSEL(SPIM_MISO, 0, 21)>;
+		};
+	};
+
+	spi1_alt_sleep: spi1_alt_sleep {
+		group1 {
+			psels = <NRF_PSEL(SPIM_SCK, 0, 19)>,
+				<NRF_PSEL(SPIM_MOSI, 0, 20)>,
+				<NRF_PSEL(SPIM_MISO, 0, 21)>;
+			low-power-enable;
+		};
+	};
+};
+
+/delete-node/ &mx25r64;
```

Comment:
Not this gives the following warning:
```
warning: NORDIC_QSPI_NOR (defined at drivers/flash/Kconfig.nordic_qspi_nor:4) was assigned the value
'y' but got the value 'n'. Check these unsatisfied dependencies: DT_HAS_NORDIC_QSPI_NOR_ENABLED
(=n).
```
Instead of removing that no longer needed `CONFIG_NORDIC_QSPI_NOR=y`, I think it will be better to rename this file to `nrf52840dk_nrf52840.overlay` and the existing `nrf52840dk_nrf52840.conf` to `nrf52840dk_nrf52840_qspi.conf`. Then, `nrf52840dk_nrf52840_spi.conf` will be no longer needed and in the default configuration for `nrf52840dk_nrf52840` this sample will use SPI for communication with the flash chip, as it is by default for all other boards. That extra test case can be then named `nrf52840dk_qspi` and use only `OVERLAY_CONFIG=boards/nrf52840dk_nrf52840_qspi.conf` in `extra_args`. The whole setup for `nrf52840dk_nrf52840` will be easier to understand. What you think?

Reply:
What you propose still requires to revert the deleted node for `sample.drivers.jesd216.nrf52840dk_qspi`